src/services/questionnaireAnalyzer/questionnaireAnalyzer.js

```diff
@@ -391,34 +391,36 @@ const checkCompletionStateOfMultipleItems = (items, props) => {
 	}
 }
 
+/**
+ * Compares to Codings for equality - assuming display is always set and always unique (as it should in all real cases)
```

Comment:
typo ? two <->to

Reply:
Yeah, a typo 😅  Should be fixed with latest commit.